Chicago-Artist-Guide/cag-backend-api
# Dev 181 cascade delete

Updated the  existing SQL tables to include cascade delete. Reset the database to include these changes and inserts. Also updated entities to include the proper join column.

How to test:
1. Run tests on main branch. This should cause a failure
2. Check out `DEV-181-CASCADE-DELETE` branch
3. run full tests will all passing
